hydromt/components/config.py

```diff
@@ -250,12 +248,23 @@ def create(
             template = self._default_template_filename
             prefix = "default"
         else:
-            self._logger.warning("No template provided and no default template found.")
+            self._logger.warning(
+                f"No default model config was found at {template}. "
+                "It wil be initialized as empty dictionary"
+            )
+            self._data = {}
+            return
+
+        if not isfile(template):
+            self._logger.warning(
+                f"Template was provided but file did not exist: {template}"
+                "It wil be initialized as empty dictionary"
+            )
+            self._data = {}
             return
 
+        template = Path(template)
         # Here directly overwrite config with template
-        if not template.is_file():
-            raise FileNotFoundError(f"Template file not found: {template}")
```

Comment:
I prefer to raise an error if the user provided a file and it's not found rather than a warning. Unless we want to add the NoData policy thing here with error by default